lib/driver.js

```diff
@@ -351,7 +351,11 @@ class IosDriver extends BaseDriver {
 
     await isolateSimulatorDevice(this.sim, this.opts);
     this.localConfig = await setLocaleAndPreferences(this.sim, this.opts, this.isSafari(), endSimulator);
-    await this.startLogCapture(this.sim);
+    if (this.caps.skipLogCapture) {
+      logger.info("'skipLogCapture' is set. Skipping starting logs, crash and system.");
```

Comment:
`Skipping starting logs, crash and system.` - this sentence looks confusing. Can we rephrase it?